src/semsql/builder/sql_schema/semsql.sql

```diff
@@ -1589,3 +1599,6 @@ CREATE VIEW transitive_edge AS WITH RECURSIVE transitive_edge
                     ON e.object = a.subject AND e.predicate = a.predicate
                )
             SELECT * FROM transitive_edge;
+
+-- patch statements table
+ALTER TABLE statements ADD COLUMN graph TEXT;
```

Comment:
This currently gets added in the `build.Makefile`. Is there a reason for not adding it to the schema?

Reply:
We wanted to avoid it going into the SQL models, because that would cause errors on older sqlite builds when using that version of semsql (e.g. in OAK). This has been out for some time so we should just put it into the schema for the next release in the minor series

Reply:
I remembered the reason. rdftab is hardwired for fewer columns, so this has to be done as postprocessing for now.